### PR TITLE
add more logging message

### PR DESCRIPTION
**Pull request by ticoann**
@amaltaro, Alan, please review.

### CODE REVIEW

File: src/python/WMComponent/ArchiveDataReporter/ArchiveDataPoller.py [hunk 1]
```diff
@@ -57,6 +57,9 @@ def algorithm(self, parameters):
                 if response[0]['status'] == "ok" and len(response[0]['ids']) == len(jobIDs):
                     for docID in jobIDs:
                         self.fwjrAPI.updateArchiveUploadedStatus(docID)
+                logging.info("...successfully uploaded %d docs", len(jobIDs))
```
Comment (amaltaro) [inline src/python/WMComponent/ArchiveDataReporter/ArchiveDataPoller.py]: These 3 lines need to go inside the `if`. Moreover, we could add an `else` to print the response returned by the server.
Typo on 'Arhcive'

Comment (ticoann) [inline src/python/WMComponent/ArchiveDataReporter/ArchiveDataPoller.py]: Thanks Alan, updated.

File: src/python/WMComponent/ArchiveDataReporter/ArchiveDataPoller.py [hunk 2]
```diff
@@ -57,6 +57,12 @@ def algorithm(self, parameters):
                 if response[0]['status'] == "ok" and len(response[0]['ids']) == len(jobIDs):
                     for docID in jobIDs:
                         self.fwjrAPI.updateArchiveUploadedStatus(docID)
+                    logging.info("...successfully uploaded %d docs", len(jobIDs))
+                    logging.debug("JobIDs uploaded:", jobIDs)
```
Comment (amaltaro) [inline src/python/WMComponent/ArchiveDataReporter/ArchiveDataPoller.py]: I always mix the need or not of the format %? Don't we need them in these two debug lines? :-D

Comment (ticoann) [inline src/python/WMComponent/ArchiveDataReporter/ArchiveDataPoller.py]: Sorry, you are a life saver. 😄

**Comment by ticoann** [comment]
test this please

**Pull request merged**